docs/_data/table/user-configuration.csv

```diff
@@ -31,7 +31,7 @@ alluxio.user.ufs.delegation.write.buffer.size.bytes,2MB
 alluxio.user.ufs.file.reader.class,alluxio.client.netty.&#8203;NettyUnderFileSystemFileReader
 alluxio.user.ufs.file.writer.class,alluxio.client.netty.&#8203;NettyUnderFileSystemFileWriter
 alluxio.user.rpc.retry.base.sleep.ms,50
-alluxio.user.rpc.retry.max.sleep.ms,5000
+alluxio.user.rpc.retry.max.sleep.ms,30000
```

Comment:
Unrelated to this PR, but should these be switched to the human readable versions? Also I find a value of `30sec` for `...sleep.ms` to be a bit misleading.